Task1.js

```diff
@@ -0,0 +1,27 @@
+function GroupByKey(array,key)
+{
+  var tempData = {};
```

Comment:
Please don't use var (only use if its extremely necessary to do so.)

Reply:
ok sir noted